man/vi.1

```diff
@@ -2185,6 +2190,13 @@ Display usage for a
 command.
 .Pp
 .It Xo
+.Cm vs Ns Op Cm plit
+.Op Ar +cmd
+.Op Ar file
+.Xc
+Split the current screen vertically.
```

Comment:
Suggest to phrase it as "Edit a different file by displaying the current screen and the new screen side by side." or something like that.

Reply:
Thanks, I iterated the wording a bit, making it more concrete by mentioning that the new screen is opened to the right of the current one. I replaced _display_ with _open_ because screens can also be _closed_, which some of the other commands do.